web/source/osk/anchoredOskView.ts

```diff
@@ -59,13 +61,37 @@ namespace com.keyman.osk {
       s.position = 'fixed';
     }
 
-    protected postKeyboardLoad() {
-      // Initializes the size of a touch keyboard.
+    public refreshLayout(pending?: boolean): void {
```

Comment:
The main difference is that I see we didn't 🍒 the @override annotation; this is reflected by three less added lines in the lines-changed count.

There's also a dropped newline at line 70 compared to the original - that's the one additional line-change difference of note.  The rest seems to match.